Index: Procfile
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>worker: java -jar ./build/libs/Banos_Bot-1.1.0-SNAPSHOT-all.jar
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Procfile b/Procfile
--- a/Procfile	(revision 2f5682a1943ccae8921ba1c722c1bae82eee51f7)
+++ b/Procfile	(date 1658082028254)
@@ -1,1 +1,1 @@
-worker: java -jar ./build/libs/Banos_Bot-1.1.0-SNAPSHOT-all.jar
\ No newline at end of file
+worker: java -jar ./build/libs/Banos_Bot-1.1.0-SNAPSHOT-all.jar.
\ No newline at end of file
